ProjectCoimbra.UWP/Project.Coimbra/Helpers/SongPagesHelper.cs

```diff
@@ -114,6 +115,36 @@ public static async Task<string> UploadSongAsync()
                 return null;
             }
 
+            return await SaveFile(file);
+        }
+
+        /// <summary>
+        /// Upload Song by Dropping File Async
+        /// </summary>
+        /// <param name="e">Drag Event</param>
+        /// <returns>Uploaded File path</returns>
+        public static async Task<string> UploadSongDropAsync(DragEventArgs e)
+        {
+            if (e.DataView.Contains(StandardDataFormats.StorageItems))
+            {
+                var items = await e.DataView.GetStorageItemsAsync();
+                if (items.Count > 0)
+                {
+                    var file = items[0] as StorageFile;
+                    return await SaveFile(file);
+                }
+            }
+
+            return null;
+        }
+
+        /// <summary>
+        /// Save file in Midi Folder
+        /// </summary>
+        /// <param name="file">File to save</param>
+        /// <returns>Saved File path</returns>
+        private static async Task<string> SaveFile(StorageFile file)
```

Comment:
By convention async method names should be suffixed with `Async` so that callers know that they need to call `await`.

So this should be called `SaveFileAsync`.